docs/introduction/what-is-conduit.mdx

```diff
@@ -4,36 +4,39 @@ sidebar_position: 0
 slug: "what-is-conduit"
 ---
 
-import BetaAlert from "../../src/components/specific-alerts/BetaAlert";
-
-Conduit is a new open-source project, written in [Go](https://go.dev/), that delivers real-time data integration for developers. It aims to provide the best user experience for building and running real-time data pipelines.
+Conduit is a new open-source project, written in [Go](https://go.dev/), that
+delivers real-time data integration for developers. It aims to provide the best
+user experience for building and running real-time data pipelines.
 
 Out of the box, Conduit comes with:
 
 - a UI
 - common connectors 
-- transforms
+- processors
 - observability
 
 The 🔋 are included.
```

Comment:
I think this line is a bit confusing 👀 took me a while to see that this is a battery